src/decoder/lattice-faster-decoder.h

```diff
@@ -42,45 +42,52 @@ struct LatticeFasterDecoderConfig {
   BaseFloat lattice_beam;
   int32 prune_interval;
   bool determinize_lattice; // not inspected by this class... used in
-                            // command-line program.
```

Comment:
These kinds of white space changes make it hard to review your code quickly. I would prefer that you not make these kinds of style changes, though it may be tempting.